api/v1/datadog/configuration_test.go

```diff
@@ -56,7 +56,7 @@ func TestConfigurationServersAccess(t *testing.T) {
 	for _, tc := range testCases {
 		t.Run(fmt.Sprintf("Index %v", tc.Index), func(t *testing.T) {
 			_, err := configuration.ServerURL(tc.Index, nil)
-			assert.Error(t, err, tc.Err)
+			assert.Error(err, tc.Err)
```

Comment:
forgot `assert := tests.Assert(ctx, t)` in this one and in the v2 version too looks like